### ISSUE TITLE
Hospitality, When guest leave they leave a HUGE ammount of gifts behind to the point of crashing the game nearly. 

### ISSUE THREAD

**Issue by VanoraBlackheart** [OrionFive/Hospitality#584]
For some reason when guest leave my colony they want to share there love with over ~50+ clothing items, or other items to the point the game just quiet litterly explodes on itself. Not quiet sure why this started to happen the last two days orso its been a issue on a new map. In advance sorry for the huge wall of text and info dump. But I'm rather clueless why this is even happening. 

https://i.gyazo.com/ef6c51c415b7db41e4e7925ad320b08b.jpg

While using Rimpy to sort out the load order. Overall the mods seem to play nicely with each other. The log window doesnt show any error lists other then this one ; Exception loading file at 

C:\SteamLibrary\steamapps\steamapps\workshop\content\294100\2491646797\LoadFolders.xml. Loading defaults instead. Exception was: System.Xml.XmlException: Data at the root level is invalid. Line 2, position 1.
  at System.Xml.XmlTextReaderImpl.Throw (System.Exception e) [0x00027] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlTextReaderImpl.Throw (System.String res, System.String arg) [0x00029] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlTextReaderImpl.Throw (System.String res) [0x00000] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlTextReaderImpl.ParseRootLevelWhitespace () [0x0012c] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlTextReaderImpl.ParseDocumentContent () [0x002d4] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlTextReaderImpl.Read () [0x0008c] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlLoader.LoadDocSequence (System.Xml.XmlDocument parentDoc) [0x0000d] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlLoader.Load (System.Xml.XmlDocument doc, System.Xml.XmlReader reader, System.Boolean preserveWhitespace) [0x000b4] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlDocument.Load (System.Xml.XmlReader reader) [0x0002e] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at System.Xml.XmlDocument.LoadXml (System.String xml) [0x00018] in <0f9699188f0c414ea6fb5557f5c16d15>:0 
  at Verse.DirectXmlLoader.ItemFromXmlString[T] (System.String xmlContent, System.String filePath, System.Boolean resolveCrossRefs) [0x0001a] in <00638943358e4807bd44cd7bc8f572da>:0 
UnityEngine.StackTraceUtility:ExtractStackTrace ()
Verse.Log:Error (string)
Verse.DirectXmlLoader:ItemFromXmlString<Verse.ModLoadFolders> (string,string,bool)
Verse.DirectXmlLoader:ItemFromXmlFile<Verse.ModLoadFolders> (string,bool)
Verse.ModMetaData:Init ()
Verse.ModMetaData:.ctor (Verse.Steam.WorkshopItem)
Verse.ModLister:RebuildModList ()
Verse.Steam.WorkshopItems:RebuildItemsList ()
Verse.Steam.WorkshopItems:.cctor ()
Verse.ModLister:RebuildModList ()
Verse.ModLister:.cctor ()
Verse.ModsConfig:.cctor ()
Verse.LoadedModManager:InitializeMods ()
Verse.LoadedModManager:LoadAllActiveMods ()
Verse.PlayDataLoader:DoPlayLoad ()
Verse.PlayDataLoader:LoadAllPlayData (bool)
Verse.Root/<>c:<Start>b__6_1 ()
Verse.LongEventHandler:RunEventFromAnotherThread (System.Action)
Verse.LongEventHandler/<>c:<UpdateCurrentAsynchronousEvent>b__27_0 ()
System.Threading.ThreadHelper:ThreadStart_Context (object)
System.Threading.ExecutionContext:RunInternal (System.Threading.ExecutionContext,System.Threading.ContextCallback,object,bool)
System.Threading.ExecutionContext:Run (System.Threading.ExecutionContext,System.Threading.ContextCallback,object,bool)
System.Threading.ExecutionContext:Run (System.Threading.ExecutionContext,System.Threading.ContextCallback,object)
System.Threading.ThreadHelper:ThreadStart ()

But that has nothing to do with hospitality. Now the game itself freezes when the items dorp on the floor and in the debug screen I get this error ; 

Too many iterations.
UnityEngine.StackTraceUtility:ExtractStackTrace ()
Verse.Log:Error (string)
RimWorld.ThingSetMakerByTotalStatUtility:GenerateDefsWithPossibleTotalValue (Verse.IntRange,single,System.Collections.Generic.IEnumerable`1<Verse.ThingDef>,RimWorld.TechLevel,RimWorld.QualityGenerator,System.Func`2<RimWorld.ThingStuffPairWithQuality, single>,System.Func`2<RimWorld.ThingStuffPairWithQuality, single>,System.Func`2<RimWorld.ThingStuffPairWithQuality, single>,System.Func`2<Verse.ThingDef, single>,int,single,bool,single)
RimWorld.ThingSetMaker_MarketValue:GeneratePossibleDefs (RimWorld.ThingSetMakerParams,single&)
RimWorld.ThingSetMaker_MarketValue:GeneratePossibleDefs (RimWorld.ThingSetMakerParams,single&,int)
RimWorld.ThingSetMaker_MarketValue:CanGenerateSub (RimWorld.ThingSetMakerParams)
RimWorld.ThingSetMaker:CanGenerate (RimWorld.ThingSetMakerParams)
RimWorld.ThingSetMaker_RandomOption/<>c__DisplayClass3_0:<Generate>b__0 (RimWorld.ThingSetMaker_RandomOption/Option)
System.Linq.Enumerable/WhereListIterator`1<RimWorld.ThingSetMaker_RandomOption/Option>:MoveNext ()
Verse.GenCollection:TryRandomElementByWeight<RimWorld.ThingSetMaker_RandomOption/Option> (System.Collections.Generic.IEnumerable`1<RimWorld.ThingSetMaker_RandomOption/Option>,System.Func`2<RimWorld.ThingSetMaker_RandomOption/Option, single>,RimWorld.ThingSetMaker_RandomOption/Option&)
RimWorld.ThingSetMaker_RandomOption:Generate (RimWorld.ThingSetMakerParams,System.Collections.Generic.List`1<Verse.Thing>)
(wrapper dynamic-method) RimWorld.ThingSetMaker:RimWorld.ThingSetMaker.Generate_Patch1 (RimWorld.ThingSetMaker,RimWorld.ThingSetMakerParams)
RimWorld.VisitorGiftForPlayerUtility:GenerateGifts (RimWorld.Faction,Verse.Map)
RimWorld.VisitorGiftForPlayerUtility:GiveRandomGift (System.Collections.Generic.List`1<Verse.Pawn>,RimWorld.Faction)
Hospitality.LordToil_VisitPoint:Leave ()
Hospitality.LordToil_VisitPoint:Cleanup ()
Verse.AI.Group.Lord:GotoToil (Verse.AI.Group.LordToil)
(wrapper dynamic-method) Verse.AI.Group.Transition:Verse.AI.Group.Transition.Execute_Patch1 (Verse.AI.Group.Transition,Verse.AI.Group.Lord)
Verse.AI.Group.Transition:CheckSignal (Verse.AI.Group.Lord,Verse.AI.Group.TriggerSignal)
Verse.AI.Group.Lord:CheckTransitionOnSignal (Verse.AI.Group.TriggerSignal)
Verse.AI.Group.Lord:LordTick ()
Verse.AI.Group.LordManager:LordManagerTick ()
Verse.Map:MapPostTick ()
(wrapper dynamic-method) Verse.TickManager:Verse.TickManager.DoSingleTick_Patch5 (Verse.TickManager)
Verse.TickManager:TickManagerUpdate ()
(wrapper dynamic-method) Verse.Game:Verse.Game.UpdatePlay_Patch1 (Verse.Game)
Verse.Root_Play:Update ()

This is the Crtl + f12 list created;

https://gist.github.com/17867250373faaa52cb2a91644c20cac

New added mods when this problem started to show up. 

ELIF SRTS Helicopters , Seventh Seraph Weapons, Vannila Factions Mechanoids, Project Rimfactor Revived + Drones , Rim Cities , SF Grim Reality , Perfect Pathfinds, Keep Bed Ownership , Infestations Spawn in Darkness , Perspective Trees, Windows , Dubs Mint Menus , Dubs Apparel Tweaks , Dubs Skylight. these are the mods I added latest to when I started to notice the behaviour to happen. Not sure where the problem is exactly. Eitherway current mod list; 
My mod list of mods currently in use. As copied out of Rimpy.

[Mod list length: 201]
Harmony [https://steamcommunity.com/sharedfiles/filedetails/?id=2009463077]
Core [https:// no link (local mod)]
Royalty [Official DLC] [https:// no link (local mod)]
Ideology [Official DLC] [https:// no link (local mod)]
Startup impact [https://steamcommunity.com/sharedfiles/filedetails/?id=1684944322]
SRTS Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=1845423808]
HugsLib [https://steamcommunity.com/sharedfiles/filedetails/?id=818773962]
Absolutely More Cannons [https://steamcommunity.com/sharedfiles/filedetails/?id=2504445815]
Vanilla Expanded Framework [https://steamcommunity.com/sharedfiles/filedetails/?id=2023507013]
Vanilla Furniture Expanded - Security [https://steamcommunity.com/sharedfiles/filedetails/?id=1845154007]
Absolutely More Cannons - Long Range Artillery [https://steamcommunity.com/sharedfiles/filedetails/?id=2562771639]
Ancient Fallout Armory [https://steamcommunity.com/sharedfiles/filedetails/?id=2406226869]
Animal Tab [https://steamcommunity.com/sharedfiles/filedetails/?id=712141500]
Animal Variety Coats [https://steamcommunity.com/sharedfiles/filedetails/?id=1511926373]
Animals are fun [https://steamcommunity.com/sharedfiles/filedetails/?id=2108362126]
Anti-Air Artillery [Unofficial Update] [https://steamcommunity.com/sharedfiles/filedetails/?id=2045686960]
AnyPlant [https://steamcommunity.com/sharedfiles/filedetails/?id=1622115915]
Apparello 2 [https://steamcommunity.com/sharedfiles/filedetails/?id=728381322]
Arsenal of Ideology [https://steamcommunity.com/sharedfiles/filedetails/?id=2561610570]
Arsenal of Reaction [https://steamcommunity.com/sharedfiles/filedetails/?id=2485424982]
Australden's Fusion Weapons Pack Remastered [https://steamcommunity.com/sharedfiles/filedetails/?id=2560174265]
Auto-Cut Blight [https://steamcommunity.com/sharedfiles/filedetails/?id=1885186534]
Beautiful Bodies [https://steamcommunity.com/sharedfiles/filedetails/?id=2068281501]
Better Pawn Control [https://steamcommunity.com/sharedfiles/filedetails/?id=1541460369]
Biosculpter Reworked [https://steamcommunity.com/sharedfiles/filedetails/?id=2560741422]
Blueprints [https://steamcommunity.com/sharedfiles/filedetails/?id=708455313]
Callouts [https://steamcommunity.com/sharedfiles/filedetails/?id=2362736503]
Interaction Bubbles [https://steamcommunity.com/sharedfiles/filedetails/?id=1516158345]
Callouts Extended! [https://steamcommunity.com/sharedfiles/filedetails/?id=2449104349]
Caravan Adventures [https://steamcommunity.com/sharedfiles/filedetails/?id=2558957509]
Character Editor [https://steamcommunity.com/sharedfiles/filedetails/?id=1874644848]
Colony Manager [https://steamcommunity.com/sharedfiles/filedetails/?id=715565262]
Allow Tool [https://steamcommunity.com/sharedfiles/filedetails/?id=761421485]
Command Palette [https://steamcommunity.com/sharedfiles/filedetails/?id=2073966033]
Contractors Arsenal [https://steamcommunity.com/sharedfiles/filedetails/?id=1576710993]
Craftsmanship [https://steamcommunity.com/sharedfiles/filedetails/?id=2010945849]
Defensive Positions [https://steamcommunity.com/sharedfiles/filedetails/?id=761219125]
Destroy Item [https://steamcommunity.com/sharedfiles/filedetails/?id=2423311270]
DontBlockDoor [https://steamcommunity.com/sharedfiles/filedetails/?id=1691076769]
Dub's Paint Shop [https://steamcommunity.com/sharedfiles/filedetails/?id=1579516669]
Dubs Apparel Tweaks [https://steamcommunity.com/sharedfiles/filedetails/?id=2296697286]
Dubs Bad Hygiene [https://steamcommunity.com/sharedfiles/filedetails/?id=836308268]
Dubs Mint Menus [https://steamcommunity.com/sharedfiles/filedetails/?id=1446523594]
Dubs Rimkit [https://steamcommunity.com/sharedfiles/filedetails/?id=832333531]
Dubs Skylights [https://steamcommunity.com/sharedfiles/filedetails/?id=833899765]
Expanded Prosthetics and Organ Engineering - Forked [https://steamcommunity.com/sharedfiles/filedetails/?id=1949064302]
EPOE-Forked: Royalty DLC expansion [https://steamcommunity.com/sharedfiles/filedetails/?id=2008970276]
Vanilla Achievements Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=2288125657]
Vanilla Fishing Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=1914064942]
ReGrowth: Core and Framework [https://steamcommunity.com/sharedfiles/filedetails/?id=2260097569]
Vanilla Plants Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=2134308522]
Farmland [https://steamcommunity.com/sharedfiles/filedetails/?id=2544524336]
FemaleBB BodyType Support [https://steamcommunity.com/sharedfiles/filedetails/?id=1792668502]
FemaleBB BodyType for Human [https://steamcommunity.com/sharedfiles/filedetails/?id=1792668839]
UN-Colony (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2019774245]
Fences And Floors (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2012420113]
Graphics Settings+ [https://steamcommunity.com/sharedfiles/filedetails/?id=1678847247]
JecsTools [https://steamcommunity.com/sharedfiles/filedetails/?id=932008009]
Giddy-up! Core [https://steamcommunity.com/sharedfiles/filedetails/?id=1216999901]
Giddy-up! Battle Mounts [https://steamcommunity.com/sharedfiles/filedetails/?id=1217001091]
Giddy-up! Caravan [https://steamcommunity.com/sharedfiles/filedetails/?id=1253251478]
Glass+Lights [https://steamcommunity.com/sharedfiles/filedetails/?id=826153738]
Gloomy Hair mk2 [https://steamcommunity.com/sharedfiles/filedetails/?id=1870678335]
Greenworld [https://steamcommunity.com/sharedfiles/filedetails/?id=1268832971]
Guards For Me [https://steamcommunity.com/sharedfiles/filedetails/?id=1855885448]
HD Pawn Rendering [https://steamcommunity.com/sharedfiles/filedetails/?id=2538746878]
High Tech Laboratory Facilities (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=1542048177]
Dual Wield [https://steamcommunity.com/sharedfiles/filedetails/?id=1628211313]
Yayo's Combat 3 [Adopted] [https://steamcommunity.com/sharedfiles/filedetails/?id=2513695736]
Humanoid Alien Races [https://steamcommunity.com/sharedfiles/filedetails/?id=839005762]
More Faction Interaction (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2379076640]
Hospitality [https://steamcommunity.com/sharedfiles/filedetails/?id=753498552]
Increased Stack [https://steamcommunity.com/sharedfiles/filedetails/?id=798158924]
Infestations Spawn in Darkness [https://steamcommunity.com/sharedfiles/filedetails/?id=2017514331]
Keep Bed Ownership [https://steamcommunity.com/sharedfiles/filedetails/?id=2130184293]
Lag Free Lamps Revamped [https://steamcommunity.com/sharedfiles/filedetails/?id=2554494500]
Let's Have a Cat! [https://steamcommunity.com/sharedfiles/filedetails/?id=1629060163]
Map Designer [https://steamcommunity.com/sharedfiles/filedetails/?id=2111424996]
Map Reroll [https://steamcommunity.com/sharedfiles/filedetails/?id=761315214]
Medical Tab [https://steamcommunity.com/sharedfiles/filedetails/?id=715565817]
Misc. Training [https://steamcommunity.com/sharedfiles/filedetails/?id=717575199]
Mod Manager [https://steamcommunity.com/sharedfiles/filedetails/?id=1507748539]
More Vanilla Turrets 1.1+ [https://steamcommunity.com/sharedfiles/filedetails/?id=2009766414]
Mortar Accuracy [https://steamcommunity.com/sharedfiles/filedetails/?id=1729446857]
Natural Paths [https://steamcommunity.com/sharedfiles/filedetails/?id=2008833499]
Natural Farms [https://steamcommunity.com/sharedfiles/filedetails/?id=2558536276]
Natural Lakes [https://steamcommunity.com/sharedfiles/filedetails/?id=2010793247]
Nature is Beautiful [1.3+] [https://steamcommunity.com/sharedfiles/filedetails/?id=2039219077]
No Lazy Doctors [https://steamcommunity.com/sharedfiles/filedetails/?id=2476396157]
Old Genesis retexture [https://steamcommunity.com/sharedfiles/filedetails/?id=2514550466]
Optimization: Meats - C# Edition [https://steamcommunity.com/sharedfiles/filedetails/?id=2542931556]
PLA-Armory [https://steamcommunity.com/sharedfiles/filedetails/?id=2439966011]
Pawn Rules [https://steamcommunity.com/sharedfiles/filedetails/?id=1499843448]
Perfect Pathfinding [https://steamcommunity.com/sharedfiles/filedetails/?id=2341486509]
Perspective Doors [https:// no link (local mod)]
Perspective Trees [https:// no link (local mod)]
[KV] RimFridge [https://steamcommunity.com/sharedfiles/filedetails/?id=1180721235]
LWM's Deep Storage [https://steamcommunity.com/sharedfiles/filedetails/?id=1617282896]
[JDS] Simple Storage [https://steamcommunity.com/sharedfiles/filedetails/?id=1961098970]
Pick Up And Haul [https://steamcommunity.com/sharedfiles/filedetails/?id=1279012058]
Prepare Landing [https://steamcommunity.com/sharedfiles/filedetails/?id=1095331978]
Princess Connect! Re:dive Summer Dress [https://steamcommunity.com/sharedfiles/filedetails/?id=2511174685]
Prisoner Harvesting [https://steamcommunity.com/sharedfiles/filedetails/?id=730854291]
Project RimFactory Revived [https://steamcommunity.com/sharedfiles/filedetails/?id=2033979700]
Project RimFactory - Drones [https://steamcommunity.com/sharedfiles/filedetails/?id=2037491557]
QualityBuilder [https://steamcommunity.com/sharedfiles/filedetails/?id=754637870]
RimForge [https://steamcommunity.com/sharedfiles/filedetails/?id=2505778174]
RIMMSqol [https://steamcommunity.com/sharedfiles/filedetails/?id=1084452457]
RIMkea [https://steamcommunity.com/sharedfiles/filedetails/?id=769201959]
RPG Style Inventory Revamped [https://steamcommunity.com/sharedfiles/filedetails/?id=2478833213]
RT Fuse [https://steamcommunity.com/sharedfiles/filedetails/?id=728314182]
ReGrowth: Boreal Forest Expansion [https://steamcommunity.com/sharedfiles/filedetails/?id=2106597000]
ReGrowth: Cold Bog Expansion [https://steamcommunity.com/sharedfiles/filedetails/?id=2182111148]
ReGrowth: Temperate Forest Expansion [https://steamcommunity.com/sharedfiles/filedetails/?id=2261114471]
ReGrowth: Tundra Expansion [https://steamcommunity.com/sharedfiles/filedetails/?id=2107412614]
ReGrowth: Vanilla Plant Density Patch [https://steamcommunity.com/sharedfiles/filedetails/?id=2270046614]
World Map Beautification Project [https://steamcommunity.com/sharedfiles/filedetails/?id=2314407956]
ReGrowth: Wastelands [https://steamcommunity.com/sharedfiles/filedetails/?id=2122461695]
Real Ruins [https://steamcommunity.com/sharedfiles/filedetails/?id=1552146295]
Realistic Planets Continued [https://steamcommunity.com/sharedfiles/filedetails/?id=2557451791]
Realistic Rooms [1.3] [https://steamcommunity.com/sharedfiles/filedetails/?id=2553724273]
Refined Floors [https://steamcommunity.com/sharedfiles/filedetails/?id=2464923082]
RenameColony [https://steamcommunity.com/sharedfiles/filedetails/?id=1359645236]
Replace Stuff [https://steamcommunity.com/sharedfiles/filedetails/?id=1372003680]
ResearchPal - Forked [https://steamcommunity.com/sharedfiles/filedetails/?id=2351393394]
RimCities [https://steamcommunity.com/sharedfiles/filedetails/?id=1775170117]
RimHUD [https://steamcommunity.com/sharedfiles/filedetails/?id=1508850027]
RimPy Mod Manager Database [https://steamcommunity.com/sharedfiles/filedetails/?id=1847679158]
Rimatomics [https://steamcommunity.com/sharedfiles/filedetails/?id=1127530465]
Roads of the Rim (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2280318231]
Royalty Tweaks [https://steamcommunity.com/sharedfiles/filedetails/?id=2018384424]
RunAndGun [https://steamcommunity.com/sharedfiles/filedetails/?id=1204108550]
RuntimeGC [1.3] [https://steamcommunity.com/sharedfiles/filedetails/?id=2559582557]
SF Grim Reality [https://steamcommunity.com/sharedfiles/filedetails/?id=1543063349]
Self Dyeing [https://steamcommunity.com/sharedfiles/filedetails/?id=2562859859]
Seventh Seraph Weapons [https:// no link (local mod)]
Simple sidearms [https://steamcommunity.com/sharedfiles/filedetails/?id=927155256]
Smarter Deconstruction and Mining [https://steamcommunity.com/sharedfiles/filedetails/?id=2398365712]
SpeakUp [https://steamcommunity.com/sharedfiles/filedetails/?id=2502518544]
Star Wars Slave Bikini Outfit [https://steamcommunity.com/sharedfiles/filedetails/?id=2014331572]
Subsurface Conduit [https://steamcommunity.com/sharedfiles/filedetails/?id=1609226728]
Suppression (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2559826227]
This Is Mine (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2560584629]
Tilled Soil [https://steamcommunity.com/sharedfiles/filedetails/?id=725747149]
Trading Control [https://steamcommunity.com/sharedfiles/filedetails/?id=2007107588]
Tribal Headgear Retexturing [https://steamcommunity.com/sharedfiles/filedetails/?id=1665073263]
UN-Furniture (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2019774854]
Use Your Gun! [https://steamcommunity.com/sharedfiles/filedetails/?id=1712993896]
VGP Vegetable Garden [https://steamcommunity.com/sharedfiles/filedetails/?id=2007061826]
Vanilla Animals Expanded — Cats and Dogs [https://steamcommunity.com/sharedfiles/filedetails/?id=1823540489]
Vanilla Apparel Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=1814987817]
Vanilla Apparel Expanded — Accessories [https://steamcommunity.com/sharedfiles/filedetails/?id=2521176396]
Vanilla Armour Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=1814988282]
Vanilla Factions Expanded - Mechanoids [https://steamcommunity.com/sharedfiles/filedetails/?id=2329011599]
Vanilla Fishing Expanded - Fishing Treasures AddOn [https://steamcommunity.com/sharedfiles/filedetails/?id=2468543398]
Vanilla Furniture Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=1718190143]
Erin's Japanese Furniture [https://steamcommunity.com/sharedfiles/filedetails/?id=2354938860]
Vanilla Furniture Expanded - Art [https://steamcommunity.com/sharedfiles/filedetails/?id=1968134023]
Vanilla Furniture Expanded - Farming [https://steamcommunity.com/sharedfiles/filedetails/?id=1957158779]
Vanilla Furniture Expanded - Medical Module [https://steamcommunity.com/sharedfiles/filedetails/?id=1718191613]
Vanilla Furniture Expanded - Production [https://steamcommunity.com/sharedfiles/filedetails/?id=1880253632]
Vanilla Hair Expanded [https://steamcommunity.com/sharedfiles/filedetails/?id=1888705256]
Vanilla Textures Expanded - Variations [https://steamcommunity.com/sharedfiles/filedetails/?id=2493234474]
Vanilla Weapons Expanded - Coilguns [https://steamcommunity.com/sharedfiles/filedetails/?id=2375692535]
Vanilla Weapons Expanded - Non-Lethal [https://steamcommunity.com/sharedfiles/filedetails/?id=2454918354]
Vault-Tec - Handy things for your vault! [https://steamcommunity.com/sharedfiles/filedetails/?id=1888778429]
Wall Light [https://steamcommunity.com/sharedfiles/filedetails/?id=1423699208]
Work Tab [https://steamcommunity.com/sharedfiles/filedetails/?id=725219116]
World Map Enhanced [https://steamcommunity.com/sharedfiles/filedetails/?id=2523248314]
Xeva Faction [https://steamcommunity.com/sharedfiles/filedetails/?id=2008120800]
Xeva Faction -XFB with Vanilla Body Patch [https://steamcommunity.com/sharedfiles/filedetails/?id=2008136195]
[ELIF] SRTS - Helicopters [https:// no link (local mod)]
[FSF] Filth Vanishes With Rain And Time [https://steamcommunity.com/sharedfiles/filedetails/?id=1508341791]
[FSF] Meditation Freedom [https://steamcommunity.com/sharedfiles/filedetails/?id=2555422930]
[JDS] EFT Apparel [https://steamcommunity.com/sharedfiles/filedetails/?id=2369524454]
[KV] Change Dresser [https://steamcommunity.com/sharedfiles/filedetails/?id=1385697110]
[KV] Faction Control [https://steamcommunity.com/sharedfiles/filedetails/?id=1509102551]
[KV] More Trait Slots [https://steamcommunity.com/sharedfiles/filedetails/?id=1508292711]
[KV] Show Hair With Hats or Hide All Hats [https://steamcommunity.com/sharedfiles/filedetails/?id=1180826364]
[NL] Facial Animation - WIP [https://steamcommunity.com/sharedfiles/filedetails/?id=1635901197]
[O21] Clutter Furniture [https://steamcommunity.com/sharedfiles/filedetails/?id=2056740389]
[O21] Role Apparel Unlocker [https://steamcommunity.com/sharedfiles/filedetails/?id=2555586497]
[RH2] BCD: CASEVAC [https://steamcommunity.com/sharedfiles/filedetails/?id=2563153311]
[RH2] BCD: First Aid [https://steamcommunity.com/sharedfiles/filedetails/?id=2563152474]
[RH2] CPERS: Arrest Here! [https://steamcommunity.com/sharedfiles/filedetails/?id=2563157350]
[RH2] Chicken Mitchell² - Hairstyles [https://steamcommunity.com/sharedfiles/filedetails/?id=2563508022]
[RH2] Hand-To-Hand [https://steamcommunity.com/sharedfiles/filedetails/?id=2556532381]
[RH2] Randy Burger - Fast Food set [https://steamcommunity.com/sharedfiles/filedetails/?id=2437337542]
[RH2] Rimdeed® - Pawn Recruitment [https://steamcommunity.com/sharedfiles/filedetails/?id=2411722784]
[RH2] Rimmu-Nation² - Clothing [https://steamcommunity.com/sharedfiles/filedetails/?id=2563506048]
[RH2] Rimmu-Nation² - Security [https://steamcommunity.com/sharedfiles/filedetails/?id=2563504596]
[RH2] Rimmu-Nation² - Weapons [https://steamcommunity.com/sharedfiles/filedetails/?id=2563507593]
[RH2] Uncle Boris' - Used Furniture [https://steamcommunity.com/sharedfiles/filedetails/?id=2563508405]
[Royalty] Unlock Working Emperor (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2457232702]
[SR]Factional War [https://steamcommunity.com/sharedfiles/filedetails/?id=2534328163]
[SR]Raid Extension [https://steamcommunity.com/sharedfiles/filedetails/?id=2554928098]
[SS]Lovely Hair Style [https://steamcommunity.com/sharedfiles/filedetails/?id=1498706339]
[sd] Advanced Powergeneration (Continued) [https://steamcommunity.com/sharedfiles/filedetails/?id=2079579039]
RimJobWorld [https:// no link (local mod)]
Rimworld-Animations [https:// no link (local mod)]
RocketMan - Performance Mod [https://steamcommunity.com/sharedfiles/filedetails/?id=2479389928]

**Comment by OrionFive** [comment]
The mod list isn't necessary. It's also in the log.

First, you should figure out which mod 2491646797 is, since it doesn't load properly, which is never a good thing.

**Comment by OrionFive** [comment]
Then check Cities and PawnRules, since they're patching the function that causes pawns not to spawn correctly in your log.

**Comment by OrionFive** [comment]
The log doesn't seem to include the incident you're talking about here, so I can't say a lot about that. But I don't think this is Hospitality related, since Hospitality is just running the vanilla gift function.

Let me know if there's anything else I can do to help.

**Comment by VanoraBlackheart** [comment]
Thank you, Will take a look at both the mods cities and pawn rules. That said the mod 24xxxxxx was SOS2 for some reason after uninstalling it kept a log behind. I removed the whole folder of the install manually. Overall I disabled the gifting for the time being and solved the problem. Sadly each time it did pop up the game just torpedoed itself so I couldnt get any logs regarding why it flooded me with the items. And if you say its vanilla's gift function there has to be another mod that messes around then with the gifting. So down the rabbit hole I go with disabling one by one to see which is the culprid. once again thank you for taking the time to look at this. If something pop ups again ill add it to this topic.
